.github/workflows/publish-extensions.yml

```diff
@@ -43,19 +54,28 @@ jobs:
       - name: Sign Production Firefox version
         continue-on-error: true
         run: yarn web-ext sign --channel=unlisted
+        # run: yarn web-ext sign --channel=listed
         env:
-          WEB_EXT_API_KEY: ${{ secrets.WEB_EXT_API_KEY }}
-          WEB_EXT_API_SECRET: ${{ secrets.WEB_EXT_API_SECRET }}
-      - uses: actions/upload-artifact@v2
-        name: Upload Artifact - Firefox Add-On XPI
-        with:
-          name: connect-addon
-          path: web-ext-artifacts/*.xpi
+          WEB_EXT_API_KEY: ${{ secrets.FIREFOX_API_KEY }}
+          WEB_EXT_API_SECRET: ${{ secrets.FIREFOX_API_SECRET }}
       - name: Get Firefox addon filename
         id: addon-file
         run: echo "::set-output name=addon_path::$(find ./web-ext-artifacts -type f -iname "*.xpi" | tail -n1)"
       - name: Rename Firefox addon file
         run: cp ${{ steps.addon-file.outputs.addon_path }} stacks-wallet-firefox.xpi
+      - name: Upload Artifact - Firefox Add-On XPI
+        uses: actions/upload-artifact@v2
+        with:
+          name: stacks-wallet-firefox
+          path: stacks-wallet-firefox.xpi
+      - name: Upload addon to Firefox store
+        uses: trmcnvn/firefox-addon@v1
+        with:
+          uuid: ${{ secrets.FIREFOX_UUID }}
+          xpi: stacks-wallet-firefox.xpi
+          manifest: manifest.json
+          api-key: ${{ secrets.FIREFOX_API_KEY }}
+          api-secret: ${{ secrets.FIREFOX_API_SECRET }}
       - name: Upload Firefox extension to GCS
```

Comment:
We can no longer do this. If we publish to the `listed` channel, then the resulting `.xpi` will not be allowed to self-install, which is why we use the `unlished` channel.

This is fine, though, since we won't use the GCS install path anymore. Let's just delete this